tests/v2/test_waku_store.nim

```diff
@@ -377,3 +377,84 @@ procSuite "Waku Store":
       decodedEmptyRes.isErr == false
       decodedEmptyRes.value == emptyRes
     
+  asyncTest "temporal history queries":
+    let
+      key = PrivateKey.random(ECDSA, rng[]).get()
+      peer = PeerInfo.init(key)
+    var
+      msgList = @[WakuMessage(payload: @[byte 0], contentTopic: ContentTopic(2)),
+        WakuMessage(payload: @[byte 1],contentTopic: ContentTopic(1), timestamp: float(1)),
```

Comment:
I thought this was a double?

Reply:
Yeah, float (float64 which is the default in Nim) is equivalent to double in protobuf types https://developers.google.com/protocol-buffers/docs/proto3#scalar